pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR/files/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.targets

```diff
@@ -0,0 +1,14 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildDependsOn>
+      CheckCoreRuntimeCompatibility;
+      $(BuildDependsOn);
+    </BuildDependsOn>
+  </PropertyGroup>
+
+  <Target Name="CheckCoreRuntimeCompatibility">
+    <Error
```

Comment:
I don't know enough about the VS targets to know if this is the right condition or not. We just need to test it.